app/models/user.rb

```diff
@@ -22,4 +22,8 @@ def self.from_omniauth(auth)
       user.skip_confirmation!
     end
   end
+
+  def can_unlock?(_rule)
```

Comment:
Will the roles able to unlock be different than those that can lock? If not I think this method would be better named as `can_manage_rule_lock?` and be used for checking permissions on lock and unlock